easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -44,6 +44,7 @@
 from vsc.utils.patterns import Singleton
 
 import easybuild.tools.environment as env
+from easybuild.tools.autopep8 import fix_code
```

Comment:
this introduces an external dependency on `pep8`, and since `autopep8` is available via PyPI (and actively maintained, it seems), I'm not a fan of copy-pasting it here as you did

in fact, I feel it's probably overkill for auto-formatting easyconfigs...